java-vtl-script/src/main/java/no/ssb/vtl/script/operations/join/AbstractJoinOperation.java

```diff
@@ -390,6 +395,25 @@ protected DataStructure computeDataStructure() {
         return newDataStructure.build();
     }
 
+    /** Checks if component is unique among other datasets  */
+    private boolean componentIsUnique(String datasetName, Map.Entry<String, Component> componentEntry) {
+        for (String otherDatasetName : datasets.keySet()) {
+            if (!Objects.equals(datasetName, otherDatasetName)) {
```

Comment:
Objects.equals() supports nulls. There is no reason the dataset name could be null, and if it is we should know about it.

Reply:
That's a good point.